### PR TITLE
[IMP] runbot_travis2docker: Adding new field Char to store list of languages

### PR DESCRIPTION
**Pull request by JesusZapata**
This PR adding one new Field named weblate_languages what contains the iso code of languages supported in the repository

image

### CODE REVIEW

**Comment by JesusZapata** [comment]
@moylop260 
This PR only have one text field in another PR I can make one widget for make the text field like the tags select

image

File: runbot_travis2docker/models/runbot_repo.py [hunk 1]
```diff
@@ -23,6 +24,19 @@ class RunbotRepo(models.Model):
...
+                                    " en_US,es_ES")
+
+    @api.multi
+    @api.constrains('weblate_languages')
+    def _check_weblate_languages(self):
+        supported_langs = [item[0] for item in scan_languages()]
+        for record in self:
+            langs = record.weblate_languages.split(',')
+            for lang in langs:
+                if lang not in supported_langs:
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_repo.py]: lang = lang.strip()

File: runbot_travis2docker/models/runbot_build.py [hunk 2]
```diff
@@ -178,6 +178,11 @@ def get_docker_run_cmd(self):
                 '-e', ('WEBLATE_HOST=%s' %
                        build.branch_id.repo_id.weblate_url)
             ]
+            if build.branch_id.repo_id.weblate_languages:
+                wl_cmd_env += [
+                    '-e', 'LANG_ALLOWED=%s' %
```
Comment (moylop260) [inline runbot_travis2docker/models/runbot_build.py:183]: Question:
MQT support use this parameter for just sync this items but what about if there is other .po... we need change MQT to process this list plus current po translations.

Comment (JesusZapata) [reply runbot_travis2docker/models/runbot_build.py:183]: @moylop260 
MQT find all files `*.po` into the folder `i18n` https://github.com/Vauxoo/maintainer-quality-tools/blob/master/travis/travis_weblate.py#L111 and add all files found in https://github.com/Vauxoo/maintainer-quality-tools/blob/master/travis/travis_weblate.py#L137

File: runbot_travis2docker/models/runbot_repo.py (hunk 1 above)
Comment (JesusZapata) [reply runbot_travis2docker/models/runbot_repo.py]: @moylop260 
Done, Thanks for you comment

**Pull request merged**